### PR TITLE
NF: `DismissNotes`'s `actualTask` return whether it succeeded

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Indeed, it always returned one out of two values. Using false or true is nicer

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java [hunk 1]
```diff
@@ -729,9 +735,9 @@ public DismissNotes(List<Long> cardIds) {
             try {
                 col.getDb().getDatabase().beginTransaction();
                 try {
-                    PairWithBoolean<Card[]> ret = actualTask(col, collectionTask, cards);
-                    if (ret != null) {
-                        return ret;
+                    boolean succeeded = actualTask(col, collectionTask, cards);
+                    if (! succeeded ) {
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java:739]:
```suggestion
                    if (!succeeded ) {
```

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
miniscule nit

File: AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java (hunk 1 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java]: Done

**Review by david-allison-1 — APPROVED** [review]
LGTM

**Review by mikehardy — APPROVED** [review]
Yeah, a boolean is >> a null when possible, nice

**Comment by Arthur-Milchior** [comment]
I just checked in Python. `True > None` raise an exception

**Pull request merged**